mythril/support/support_args.py

```diff
@@ -0,0 +1,13 @@
+from mythril.support.support_utils import Singleton
+
+
+class Args(object, metaclass=Singleton):
```

Comment:
I wouldn't have this be a singleton, this makes running multiple analysis within one process more complex